types/wlr_layer_shell_v1.c

```diff
@@ -498,7 +509,7 @@ struct wlr_layer_shell_v1 *wlr_layer_shell_v1_create(struct wl_display *display)
 	}
 
 	struct wl_global *global = wl_global_create(display,
-		&zwlr_layer_shell_v1_interface, 2, layer_shell, layer_shell_bind);
```

Comment:
Hm, wierd, we only advertise v2? A destroy request was added in v3: https://github.com/swaywm/wlr-protocols/commit/23490a04798bd0ee5d927f3f717087ded416f391

It seems like we implemented the new request, but forgot to bump the version: https://github.com/swaywm/wlroots/commit/45c0877e3433f2b057f9125a8bc735057e088e6c#diff-8feb2aaa9e09a85d35f888167858b8c51abb54a3dcd2e676279a8c4a5d67936c

cc @ifreund

Reply:
Good catch: #2608